.travis.yml

```diff
@@ -576,29 +576,38 @@ matrix:
 
     - os: linux
       name: R Package
-      if: (type = push AND branch = master) OR type = pull_request OR tag =~ /^v\d+\.\d+\.\d+$/
+      if: (type = push) OR type = pull_request OR tag =~ /^v\d+\.\d+\.\d+$/
```

Comment:
any reason the master check has disappeared?

Reply:
The net effect is that R is now also checked for every branch, even if it doesn't have a pull request yet. However, R is lined up last and needs to wait anyway. Maybe we can shift it up in the build matrix, or disable altogether?

Reply:
I'm okay running it all the time but then just delete the if line altogether

Reply:
This was meant to be a temporary change, just to get started with the R build in a branch without a PR. One R build for non-PR builds might be ok, the full suite can only be run on master then?